backend/src/modules/doubles-tournament/core/application/event/StartMatchAfterItsCallingEventHandler.ts

```diff
@@ -0,0 +1,20 @@
+import { EventHandler } from '../../../../../shared/core/application/event/EventHandler';
+import { MatchWasCalled } from '../../domain/event/MatchWasCalled';
+import { CommandPublisher } from '../../../../../shared/core/application/command/CommandBus';
+import { StartMatch } from '../../../../match-module/core/application/command/StartMatch';
+import { MatchId } from '../../domain/MatchId';
+
+export class StartMatchAfterItsCallingEventHandler implements EventHandler<MatchWasCalled> {
```

Comment:
Jak dla mnie to tego EventHandler nie musi być w nazwie :) 

StartMatchAfterMatchWasCalled / MatchWasCalledCausesStartMatch / StartMatchWhenMatchWasCalled by było spoko :) I od razu widać jasno jaki event jaką command.

Reply:
@nowakprojects  - a dlaczego tak? Pytam dla wiedzy, nie kwestionuję.
Moje wątpliwości:
Nie łatwiej jest się odnaleźć (choć moze tylko dla mnie) jak podpisany jest z EvendHandler w nazwie? 
No i łatwiej jest znaleźć w stercie plików. nie trzeba wszystkich szukać tylko frazy szukasz w nazwie

Reply:
Trochę tak, ale w sumie nazwa sugeruje co dokładnie jest robione :-), więc patrząc po plikach albo wpisując nazwę typu coś z Match albo Email znajdziesz szybko w danym folderze